backend/src/main.py

```
@@ -1,13 +1,71 @@
+import asyncio
+import logging
+from contextlib import asynccontextmanager
+
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 
-from src.api import api_router
+from src.database.core import Base, engine
+
+# Import routers
+from src.base_router import base_router 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)